src/wrapper/java_vm/vm.rs

```diff
@@ -44,6 +44,11 @@ impl JavaVM {
         Ok(JavaVM(ptr))
     }
 
+    /// Returns underlying `sys::JavaVM` interface.
+    pub fn get_native_interface(&self) -> *mut sys::JavaVM {
```

Comment:
I think 'native interface' might be confused with the JNI interface pointer (JNIEnv): https://docs.oracle.com/javase/10/docs/specs/jni/design.html#jni-interface-functions-and-pointers

I wonder if anyone can come up with a better name. I can think of `get_raw_vm_pointer` or `get_java_vm_pointer`?